src/Shared/Microsoft.AspNetCore.Razor.Utilities.Shared/DictionaryExtensions.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Razor;
```

Comment:
Change namespace to `System.Collections.Generic` as we've done for other extension methods in `Shared.Utilties` to facilitate discoverability.